servers/src/common/adapters.rs

```diff
@@ -326,10 +326,20 @@ impl p2p::ChainAdapter for NetToChainAdapter {
 		total_size: u64,
 	) -> bool {
 		match self.sync_state.status() {
-			SyncStatus::TxHashsetDownload { .. } => {
+			SyncStatus::TxHashsetDownload {
```

Comment:
Doesn't a `, ..` at the end work here for fields you don't care about? Instead of specifying them explicitly.

Reply:
👍 Updated